### PR TITLE
more cards 5

### PR DESCRIPTION
**Pull request by bilbo2**
Add more new cards after playtesting (credit goes to Kelsioboy):
    Akuta, Born of Ash #1426 
    Dark Dabbling #1299 
    Defiling Tears #1300 
    Doorkeeper #1386 
    Dragon Grip #1376 
    Explorer's Scope #1382 
    Gigapede #1437 
    Guul Draz Overseer #1372
    Into the Wilds #1383 
    Salt Road Ambushers #1378
    Tainted Remedy #1379 
    Terrifying Presence #1366 
    Thrasios, Triton Hero #1389 
    Trail of Mystery #1381 
    Yomiji, Who Bars the Way #1373

### CODE REVIEW

File: release/Magarena/scripts/Dark_Dabbling.txt [hunk 1]
```diff
@@ -1,11 +1,10 @@
 name=Dark Dabbling
-image=https://magiccards.info/scans/en/ori/89.jpg
+image=http://magiccards.info/scans/en/ori/89.jpg
```
Comment (melvinzhang) [inline release/Magarena/scripts/Dark_Dabbling.txt]: url should be https

File: release/Magarena/scripts/Salt_Road_Ambushers.txt [hunk 2]
```diff
@@ -1,13 +1,12 @@
 name=Salt Road Ambushers
-image=https://magiccards.info/scans/en/dtk/198.jpg
+image=http://magiccards.info/scans/en/dtk/198.jpg
```
Comment (melvinzhang) [inline release/Magarena/scripts/Salt_Road_Ambushers.txt]: should use https

File: release/Magarena/scripts/Trail_of_Mystery.txt [hunk 3]
```diff
@@ -1,11 +1,10 @@
 name=Trail of Mystery
-image=https://magiccards.info/scans/en/ktk/154.jpg
+image=http://magiccards.info/scans/en/ktk/154.jpg
```
Comment (melvinzhang) [inline release/Magarena/scripts/Trail_of_Mystery.txt]: use https

**Comment by bilbo2** [comment]
Fixed for all cards with the same problem (using http for magiccards.info)

**Pull request merged**